client/component/src/AnyPlot.tsx

```diff
@@ -329,6 +329,20 @@ function AnyPlot(props: AnyPlotProps) {
     interactionTime.current = interaction.end();
   });
 
+  if (props.batonProps === null) {
```

Comment:
Should this be a test for undefined? I.e. a falsify check `if (!props.batonProps) {`